app/assets/javascripts/select-kit/addon/components/bulk-select-topics-dropdown.js

```diff
@@ -0,0 +1,67 @@
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import ChangeCategory from "discourse/components/bulk-actions/change-category";
+import BulkTopicActions from "discourse/components/modal/bulk-topic-actions";
+import TopicBulkActions from "discourse/components/modal/topic-bulk-actions";
+import DropdownSelectBoxComponent from "select-kit/components/dropdown-select-box";
+
+export default DropdownSelectBoxComponent.extend({
+  classNames: ["bulk-select-topics-dropdown"],
+  headerIcon: null,
+  showFullTitle: true,
+  selectKitOptions: {
+    showCaret: true,
+    showFullTitle: true,
+    none: "select_kit.components.bulk_select_topics_dropdown.title",
+  },
+
+  modal: service(),
+
+  computeContent() {
+    let options = [];
+    options = options.concat([
+      {
+        id: "update-category",
+        icon: "pencil-alt",
+        name: "Update Category",
+        description: "Choose the new category for the selected topics",
```

Comment:
This and the other `description` and `name` need to be localized, probably a new `topic_bulk_actions:` key to `client.en.yml` for this